### ISSUE TITLE
Invalid webpack configuration for Windows

### ISSUE THREAD

**Issue by coreylenertz** [Webiny/webiny-js#516]
Webiny Version: master
OS Platform: Windows

`$ packages\admin> yarn start`

## Problem:
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.module.rules[2].oneOf[1].include should be one of these:
   RegExp | string | function | [(recursive)] | object { and?, exclude?, include?, not?, or?, test? } | [RegExp | string | function | [(recursive)] | object { and?, exclude?, include?, not?, or?, test
? }]

The rule's include is an array of string that contain incorrect path separators.

## Workaround:
My work around was to correct that paths after building the newConfig but before returning it to webpack.
```
const rule = newConfig.module.rules[2].oneOf[1]
rule.include = rule.include.map(inc => ({ include: inc.replace(/\//, "\\") }))
```
https://github.com/Webiny/webiny-js/blob/f6a85d725e4ae172233debc5db3f493b41481b05/packages/webiny-cli/src/init/template/packages/webiny-rewire/configs/index.js#L19

Obviously, not a solid solution for all. I haven't looked up the chain to see why the paths are incorrect from the start.

**Comment by Pavel910** [comment]
@coreylenertz Great, thanks for the report, I'll look into it!

**Comment by Pavel910** [comment]
@coreylenertz dived into this one a bit deeper, turns out the package `get-yarn-workspaces` we depend on is always returning paths with forward slash. I added some extra handling for that, will be in the next release.

Thanks for reporting, cheers!

**Comment by Pavel910** [comment]
This is fixed and released in webiny-cli@2.2.0. Closing.
